packages/react/radio-group/src/RadioGroup.tsx

```diff
@@ -73,24 +76,27 @@ const RadioGroup = React.forwardRef((props, forwardedRef) => {
       name,
       value,
       required,
+      rovingFocus,
       onValueChange: composeEventHandlers(handleValueChange, (event) => {
         setValue(event.target.value);
       }),
     }),
-    [name, value, required, handleValueChange, setValue]
+    [name, value, required, rovingFocus, handleValueChange, setValue]
+  );
+
+  const primitive = (
+    <Primitive
+      {...groupProps}
+      selector={selector}
+      ref={forwardedRef}
+      role="radiogroup"
+      aria-labelledby={labelledBy}
+    />
   );
 
   return (
     <RadioGroupContext.Provider value={context}>
-      <RovingFocusGroup loop>
-        <Primitive
-          {...groupProps}
-          selector={selector}
-          ref={forwardedRef}
-          role="radiogroup"
-          aria-labelledby={labelledBy}
-        />
-      </RovingFocusGroup>
+      {rovingFocus ? <RovingFocusGroup loop>{primitive}</RovingFocusGroup> : primitive}
```

Comment:
I was torn between doing this or adding a `ConditionalWrap` component. Which do we prefer?

Reply:
Historically we've stuck with this for now. I'm happy to keep this.